Sources/TuistKit/Commands/TrackableCommand/TrackableCommand.swift

```diff
@@ -25,19 +26,22 @@ public class TrackableCommand: TrackableParametersDelegate {
     private let commandArguments: [String]
     private let commandEventFactory: CommandEventFactory
     private let asyncQueue: AsyncQueuing
+    private let fileHandler: FileHandling
```

Comment:
Should we take the opportunity to use `FileSystem` here?

Reply:
I don't think `FileSystem` has a `currentPath` reference. Or at least I couldn't find it.